_posts/2015-01-02-topics.md

```diff
@@ -9,36 +9,45 @@ icon-title: true
 
 ### Topics of interest include, but are not limited to:
 
-- Architecture and design of HPC and Big Data systems
-- Architecture and design of ETL and data acquisition pipelines
+- Architecture and design of High Performance Computing (HPC) and Big Data systems
+- Architecture and design of ETL (Extract, Transform and Load) and data acquisition pipelines
```

Comment:
Isn't it more consistent with the above to say  `Extract Transform and Load (ETL)` ?